perfrepo-cli

```diff
@@ -157,18 +157,18 @@ class TestCLI(GenericCLI):
             try:
                 test_id = int(argv[1])
                 test = self._perf_api.test_get_by_id(test_id)
-                if test is None:
-                    test = self._perf_api.test_get_by_uid(test_id)
-                if test is None:
-                    print("Test not found.")
-                else:
-                    print(test)
             except IndexError:
                 print("No ID specified!", file=sys.stderr)
                 return -1
             except ValueError:
-                print("ID needs to be an integer!", file=sys.stderr)
-                return -1
+                test = self._perf_api.test_get_by_uid(argv[1])
```

Comment:
I think this is nonsense. Despite that it is possible to create a test with numeric UID in PerfRepo (just have tested it), users should not do this. Because with every test created the ID increments. Imagine you have test with UID=100. And perfrepo also contains a test with ID=100. Which one it should return? I think that we should really distinguish between numeric and non numeric values specified.